main.py

```diff
@@ -308,6 +310,10 @@ def run(mgr, args, tenderly_uri=None) -> None:
     start_timeout = time.time()
     mainnet_uri = mgr.cfg.w3.provider.endpoint_uri
     handle_static_pools_update(mgr)
+
+    pool_finder = PoolFinder(uni_v2_forks=mgr.cfg.network.UNI_V2_FORKS, uni_v3_forks=mgr.cfg.network.UNI_V3_FORKS, solidly_v2_forks=mgr.cfg.network.SOLIDLY_V2_FORKS, carbon_forks=mgr.cfg.network.CARBON_V1_FORKS, flashloan_tokens=args.flashloan_tokens) if args.pool_finder != -1 else None
+    if pool_finder:
```

Comment:
Why not just do a simple if, then create and run the path finder, instead of doing an 'if', and then another 'if' based on the result of the previous 'if'?

Reply:
Fixed